frontend/app/services/tour.js

```diff
@@ -159,8 +159,8 @@ export default class TourService extends Tour {
               await this.router.transitionTo(
                 this.autostartTour.undoneTours.shift() ?? "index"
               );
-            } catch (error) {
-              this.notify.error("unexpected error");
+            } catch {
+              /* tslint:disable:no-empty */
```

Comment:
tslint? AFAIK we use eslint, and no typescript :c

Reply:
changed, Thanks

Reply:
true dat.. `// eslint-disable-next-line no-empty`